protobackend/output.py

```diff
@@ -55,10 +59,15 @@ def wrapper(*args, **kwargs):
         with CaptureErrors(fallback_output):
             output = f(*args, **kwargs)
             if not isinstance(output, list):
+                if output is not None:
+                    warnings.warn(
+                        "Only lists are valid return values for executed commands"
```

Comment:
Here I guess `only lists and None` is more appropriate?

Reply:
maybe `only lists and (implicit) None`?

Reply:
This comment is actually linked to the other one. So in Python `return None` or no return gives the same output? If so, maybe `"Executed commands are expected to return a list or nothing"`. Do we want to mention the output has been stripped out to an empty array?

Reply:
Yes, that's what Python does! I updated the message.